packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -103,3 +103,26 @@
 		display: block;
 	}
 }
+
+// Show a draggable handle when you're dragging using the toolbar component.
+.components-draggable__clone {
```

Comment:
How can we make this style specific to the block-toolbar  draggable?

Reply:
I can't think of a way :( — the element has to be attached to the thing you are dragging around, which is the draggable clone element.

Reply:
I pushed a fix for this.